pleskdistup/common/src/systemd.py

```diff
@@ -24,6 +24,35 @@ def is_service_active(service: str):
     return res.returncode == 0
 
 
+def get_required_services(service: str) -> typing.List[str]:
+    res = subprocess.run(
+        [SYSTEMCTL_BIN_PATH, 'cat', service],
```

Comment:
I doubt that is reliable. For example, one can create dependencies by putting symlinks into a `<service>.requires` directory instead of declaring them in backing files. Also, there's the reverse form — `RequiredBy` and alternatives, like `Requisite`.
Likely, you should use `systemctl show --property` instead.